src/libs/Accessibility/index.ts

```diff
@@ -1,25 +1,26 @@
 import {useEffect, useState, useCallback} from 'react';
-import {AccessibilityInfo} from 'react-native';
-import _ from 'underscore';
+import {AccessibilityInfo, LayoutChangeEvent} from 'react-native';
 import moveAccessibilityFocus from './moveAccessibilityFocus';
 
-const useScreenReaderStatus = () => {
+const useScreenReaderStatus = (): boolean => {
     const [isScreenReaderEnabled, setIsScreenReaderEnabled] = useState(false);
     useEffect(() => {
         const subscription = AccessibilityInfo.addEventListener('screenReaderChanged', setIsScreenReaderEnabled);
 
-        return subscription && subscription.remove;
+        return () => {
+            subscription?.remove();
+        };
     }, []);
 
     return isScreenReaderEnabled;
 };
 
-const getHitSlopForSize = ({x, y}) => {
+const getHitSlopForSize = ({x, y}: {x: number; y: number}) => {
```

Comment:
Just a question: do we really need an object here? I see that we use this function once and puts an object. It looks strange, because across the project we prefer list of arguments to an object.
It is up to you here.
```suggestion
const getHitSlopForSize = (x: number, y: number) => {
```

Reply:
I see your point and I am ok with that change, but I am not sure if we should do such refactors during TS migration. 
@fabioh8010 WYDT?

Reply:
I think it's out of the scope of the migration. But if we were to standardise on one way, I'd argue objects would be better so we don't end up with giant lists of parameters like in `OptionsListUtils.getNewChatOptions`

Reply:
@thiagobrez I agree with you 100%. From my understanding up to 3 arguments we can have the list, more than 3 - an object of arguments.

Reply:
In this case we agree that it could have been implemented with 2 parameters vs 1 object, but if we change that we need to refactor other places and I also think is out of scope.